tests/cli/test_main_clean.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+
+from glob import glob
+from json import loads as json_loads
+from os import listdir
+from os.path import basename, isdir, join, exists
+from shutil import copy
+import pytest
+from conda.base.constants import CONDA_TEMP_EXTENSION
+from conda.core.subdir_data import create_cache_dir, SubdirData
+from conda.testing.integration import make_temp_package_cache, run_command, Commands, make_temp_env
+
+
+def _get_contents(pkgs_dir):
+    return [join(pkgs_dir, d) for d in listdir(pkgs_dir)]
+
+
+def _get_pkgs(pkgs_dir=None, *, contents=None):
+    return [d for d in contents or _get_contents(pkgs_dir) if isdir(d)]
+
+
+def _get_tars(pkgs_dir=None, *, contents=None):
+    return [t for t in contents or _get_contents(pkgs_dir) if t.endswith((".tar.bz2", ".conda"))]
```

Comment:
```suggestion
def _get_contents(pkgs_dir):
    return [join(pkgs_dir, name) for name in listdir(pkgs_dir)]


def _get_pkgs(pkgs_dir=None, *, contents=None):
    return [content for content in contents or _get_contents(pkgs_dir) if isdir(content)]


def _get_tars(pkgs_dir=None, *, contents=None):
    return [content for content in contents or _get_contents(pkgs_dir) if content.endswith((".tar.bz2", ".conda"))]
```

Reply:
I mean, this is just a minor nit, simplifies debugging maybe in the future.

Reply:
Ah, this comes from a personal convention where if the list/dict comprehension fits on a single line I'll use one-char variables to keep it super short and sweet but I agree it produces inconsistencies so I'll make the change.

Reply:
Done

Reply:
Thank you!